app/frontend/components/bookWorkHourModal.tsx

```diff
@@ -83,7 +82,7 @@ export const BookWorkHourModal = (props: BookWorkHourModalProps): JSX.Element =>
           </select>
         </label>
         <label>
-          <select className="w-72 rounded-md" {...register('taskId')}>
+          <select className="w-72 rounded-md" {...(register('taskId'), { required: true })}>
```

Comment:
I think adding `required ` is causing problems in loading the tasks in the input. I might of course be missing something, though.